### DISCUSSION TITLE
Change audio track order

### DISCUSSION THREAD

**Discussion by Ninko85** [mifi/lossless-cut#1589] [Q&A]
Hi,
Is it possible to change the order the audio tracks are in?

Thanks

**Comment by mifi** [comment]
I'm not entirely sure how to do that with ffmppeg, but if it's possible, then maybe it could be done. maybe as a workaround you could export each track as a separate file. then drag them back in one by one in the order you want, and export your file with all the imported tracks

**Reply by Ninko85** [reply]
That's the only way I could think of too, but I just thought I'd ask incase I was missing something.

Thanks

**Reply by Ninko85** [reply]
After searching, does this help @mifi 

https://video.stackexchange.com/questions/29233/change-default-audio-track-in-mp4-video

**Reply by mifi** [reply]
Ok cool. I think the code in losslesscut that handles tracks and editing is quite complex so I'm not sure how much work to add this, but maybe if it's a popular feature request, then it can be implemented some day

**Reply by ror3d** [reply]
Is there an issue to upvote for this, or should that be done just in the question itself?

**Reply by mifi** [reply]
I've created an issue #1942 - you can leave a thumbs up there. it also describes a workaround for how to do it manually.

**Comment by Ninko85** [comment]
Can this be done by editing the track metadata?
I tried adding the tag value "index" and adding the correct order numbers, but after exporting it didn't make any changes.
Any thoughts?

**Reply by mifi** [reply]
No, that won't work unfortunately.
